UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ValidateAuthorizationForAPIsWithExchangeSetStandardParameterTests.cs

```diff
@@ -23,102 +22,79 @@ public void SetupAsync()
         // PBI 140109 : ESS API : Add authorization to allow only UKHO people to create unencrypted ES 
         [Test]
         [Category("SmokeTest-AIODisabled")]
-        public async Task WhenICallTheSinceDateTimeApiWithAValidB2cTokenAndIsUnencryptedParameterAsTrue_ThenAnUnauthorizedResponseIsReturned()
+        public async Task WhenICallTheSinceDateTimeApiWithAValidB2cTokenAnds57AsexchangeSetStandardParameter_ThenAnUnauthorizedResponseIsReturned()
         {
-            var apiResponse = await ExchangeSetApiClient.GetExchangeSetBasedOnDateTimeAsync(sinceDateTime, null, accessToken: EssB2CToken, "True");
+            var apiResponse = await ExchangeSetApiClient.GetExchangeSetBasedOnDateTimeAsync(sinceDateTime, null, accessToken: EssB2CToken, "s57");
             Assert.AreEqual(401, (int)apiResponse.StatusCode, $"Incorrect status code {apiResponse.StatusCode} is returned, instead of the expected 401.");
         }
 
         // PBI 140109 : ESS API : Add authorization to allow only UKHO people to create unencrypted ES 
         [Test]
         [Category("QCOnlyTest-AIODisabled")]
-        public async Task WhenICallTheSinceDateTimeApiWithAValidADTokenAndIsUnencryptedParameterAsTrue_ThenACorrectResponseIsReturned()
+        public async Task WhenICallTheSinceDateTimeApiWithAValidADTokenAnds57AsexchangeSetStandardParameter_ThenACorrectResponseIsReturned()
         {
-            var apiResponse = await ExchangeSetApiClient.GetExchangeSetBasedOnDateTimeAsync(sinceDateTime, null, accessToken: EssJwtToken, "True");
+            var apiResponse = await ExchangeSetApiClient.GetExchangeSetBasedOnDateTimeAsync(sinceDateTime, null, accessToken: EssJwtToken, "s57");
             Assert.AreEqual(200, (int)apiResponse.StatusCode, $"Incorrect status code is returned {apiResponse.StatusCode}, instead of the expected 200.");
 
             //verify model structure
             await apiResponse.CheckModelStructureForSuccessResponse();
-
-            //Get the BatchId
-            var fssBatchId = await apiResponse.GetBatchId();
-            cleanUpBatchIdList.Add(fssBatchId);
         }
         #endregion
 
         #region ProductIdentifier Api
         // PBI 140109 : ESS API : Add authorization to allow only UKHO people to create unencrypted ES 
         [Test]
         [Category("SmokeTest-AIODisabled")]
-        public async Task WhenICallTheProductIdentifierApiWithAValidB2cTokenAndIsUnencryptedParameterAsTrue_ThenAnUnauthorizedResponseIsReturned()
+        public async Task WhenICallTheProductIdentifierApiWithAValidB2cTokenAnds57AsexchangeSetStandardParameter_ThenAnUnauthorizedResponseIsReturned()
```

Comment:
Please change Identifier to Identifiers wherever applicable
```suggestion
        public async Task WhenICallTheProductIdentifiersApiWithAValidB2cTokenAnds57AsexchangeSetStandardParameter_ThenAnUnauthorizedResponseIsReturned()
```

Reply:
Resolved. Thanks